R/chat.R

```diff
@@ -447,6 +448,7 @@ Chat <- R6::R6Class("Chat",
           cat_line(format(turn), prefix = "< ")
         }
       }
+      turn@completed <- Sys.time()
```

Comment:
Why do you have to update this here instead of relying on the value set by constructor called in `value_turn()` above?